### DISCUSSION TITLE
How does domain prolongation work?

### DISCUSSION THREAD

**Discussion by d47081** [Revertron/Alfis#357] [Q&A] [answered]
For example, I have mined some domain yesterday, so have it a sense to mine it today?
 
* it's +1 year of new block time or +1 year of last expiration date?
* must I select checkbox `Renew domain by using higher difficulty while mining.` - what this option means in other words?
* and this notice not clear `Starting from block 5000 the change of DNS records is decoupled from domain renewal`

Can't find any documentation

**Comment by Revertron** [comment]
If you mined your domain, and it hasn't expired, then you can:
1. Change some records for this domain (cheap mining)
2. Change records + prolongate the domain life (more mining)
3. Prolongate without changing anything (more mining, as in pt.2).

**Reply by d47081** [reply]
Thanks for reply, 

but I don't understand when it's better to run miner for due date prolongation (no. 3 case)

the mining new block will add plus one year to expiration date or just one year from the block mined time?

**Reply by Revertron** [reply, marked as answer]
You can use prolongation whenever you please. It adds a year from the time of block you mine.

**Reply by d47081** [reply]
Thank you, clear!
